src/flexmeasures_client/client.py

```diff
@@ -436,3 +436,51 @@ async def get_sensor_data(
         data_fields = ("values", "start", "duration", "unit")
         sensor_data = {k: v for k, v in response.items() if k in data_fields}
         return sensor_data
+
+    async def get_sensor(self, sensor_id: int):
+        uri = f"sensors/{sensor_id}"
+        response, status = await self.request(uri=uri, method="GET")
+        check_for_status(status, 200)
```

Comment:
Maybe we can add to the docstring that this function raises a `ValueError` exception in case of not being able to fetch the sensor.

Reply:
There are many errors this function can throw in case you use incorrect values as well. Is it relevant to mentions this error specifically?

Reply:
No need to specify all of them, IMO. I would just say that it can raise a `ValueError` exception. This way users can  can plan ahead what to do in case of not being able to fetch a senso.

Reply:
This would be a nice addition to the other methods as well.